wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -22,7 +22,24 @@
 
             <!-- Edit this section for important announcements.-->
             <div class="alert alert-warning text-center">
-                <a href="#">Important news will come here, after we add tag 'imp' with them.</a>
+                <ul id="imp_announcement">
+                    <h3>Important Announcements:</h3>
```

Comment:
Okay sir, it also needs if-else there incase there are no announcement/news, will update the PR